src/Glimpse.Server.Web/Resources/History.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Glimpse.Server.Web.Extensions;
+using Microsoft.AspNet.Http;
+using Microsoft.Net.Http.Headers;
+
+namespace Glimpse.Server.Web
+{
+    public class History : IResource
```

Comment:
We could go back to `*Resource` if that's preferred.

I removed it so that when this becomes a key in the resources metadata object, the key would be `history`, and the other resource would be `request-history`.

I can update the Skewer naming code to drop the word Resource, but thought that might lead to other problems.